spec/controllers/user_sessions_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe UserSessionsController do
+  let(:user) { create(:user, activated_at: Time.new(2007, 1, 1)) }
+
+  context "create" do
+    context "after failed to login more than consecutive login limit" do
+      before do
+        max_consecutive_attempts = 2
+        allow(UserSession).to receive(:consecutive_failed_logins_limit).and_return(max_consecutive_attempts)
+        (max_consecutive_attempts + 1).times do
```

Comment:
You don't need to add 1, this test works if you use `max_consecutive_attempts.times`.